protos/protos/public/modeldb/metadata/MetadataService.proto

```diff
@@ -90,6 +90,37 @@ message DeletePropertyRequest {
     }
 }
 
+message AddKeyValuePropertiesRequest {
+    IdentificationType id = 1;
+    string property_name = 2;
+    repeated common.KeyValue key_value = 3; // Only string value expected
+
+    message Response {
+    }
+}
+
+message GetKeyValuePropertiesRequest {
+    // If only id_type is provided, gather from all of such sources
+    IdentificationType id = 1;
+    string property_name = 2;
+    repeated string key = 3;
+    bool getAll = 4;
```

Comment:
add a comment if this will 
`get all key value corresponding to id and property`
OR 
`get all properties with their key and value for a given id`

I think it is the first one, but please add a comment

Reply:
Added comment on message